pydm/widgets/byte.py

```diff
@@ -13,10 +13,10 @@ class PyDMBitIndicator(QWidget):
         The parent widget for the Label
 
     """
-    def __init__(self, parent=None):
+    def __init__(self, circle=False, parent=None):
```

Comment:
@klauer would you mind to make `circle=False` be the last parameter? 
If someone is using this widget in the wild with code it can break in case they are not using named parameters.